Package.swift

```diff
@@ -1,151 +1,67 @@
-         let regex = try! NSRegularExpression(pattern: "\\#[\\s\\S]*?.*$", options: .anchorsMatchLines)
-         let nsString = text as NSString
-         let range = NSRange(location: 0, length: nsString.length)
-         let matches = regex.matches(in: text, range: range)
-         var trimmedCommentsText = text
-         matches.forEach {
-             trimmedCommentsText = trimmedCommentsText
-                 .replacingOccurrences(of: nsString.substring(with: $0.range), with: "")
-         }
-         return trimmedCommentsText
-     }
- }
-
- fileprivate extension Array where Element == String {
-     subscript(_ token: String) -> String? {
-         guard let tokenIndex = firstIndex(of: token),
-             count > tokenIndex,
-             !["machine", "login", "password"].contains(self[tokenIndex + 1]) else {
-                 return nil
-         }
-         return self[tokenIndex + 1]
-     }
- }
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "MapboxMaps",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v11)],
+    products: [
+        .library(
+            name: "MapboxMaps",
+            targets: ["MapboxMaps"]),
+    ],
+    dependencies: [
+        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("10.0.0-beta.11")),
+        .package(name: "MapboxCoreMaps", url: "https://github.com/mapbox/mapbox-core-maps-ios.git", .exact("10.0.0-beta.15")),
+        .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", .exact("0.10.8")),
+        .package(name: "Turf", url: "https://github.com/mapbox/turf-swift.git", from: "1.2.0"),
+    ],
+    targets: [
+        .target(
+            name: "MapboxMaps",
+            dependencies: ["MapboxCommon", "MapboxCoreMaps", "Turf", "MapboxMobileEvents"],
+            exclude: [
+                "Annotations/Info.plist",
+                "Foundation/Info.plist",
+                "Gestures/Info.plist",
+                "Location/Info.plist",
+                "MapView/Info.plist",
+                "Offline/Info.plist",
+                "Ornaments/Info.plist",
+                "Snapshot/Info.plist",
+                "Style/Info.plist",
+                "Style/README.md",
+            ]
+        ),
+        .testTarget(
+            name: "MapboxMapsTests",
+            dependencies: ["MapboxMaps"],
+            exclude: [
+                "Annotations/Info.plist",
+                "Foundation/Info.plist",
+                "Gestures/Info.plist",
+                "Location/Info.plist",
+                "MapView/Info.plist",
+                "Offline/Info.plist",
+                "Ornaments/Info.plist",
+                "Snapshot/Info.plist",
+                "Style/Info.plist",
+                "MapView/Integration Tests/HTTP/HTTPIntegrationTests.swift"
```

Comment:
```suggestion
                "MapView/Integration Tests/HTTP/HTTPIntegrationTests.swift",
```

For consistency with other arrays wrapped to multiple lines.

Reply:
Main benefit of this style is cleaner diffs as things are added/removed to the end of these arrays.